installer/installer.py

```diff
@@ -1114,7 +1138,11 @@ def install_steps():
 
     # Steps to execute if upgrading existing node
     def upgrade_steps():
-        interviewer.ask_for_cosmovisor()
+        # if cosmovisor is not installed 
+        if not os.path.exists(os.path.join(DEFAULT_INSTALL_PATH, DEFAULT_COSMOVISOR_BINARY_NAME)):
+            interviewer.ask_for_cosmovisor()
+        else:
+            interviewer.ask_for_cosmovisor_bump()
```

Comment:
If there's a Cosmovisor binary installed, this will _always_ ask to bump - even if the user already has Cosmovisor 1.3 installed. (Remember that someone might run the upgrade more than once.)

We don't need to do this. You can actually check the version of Cosmovisor running using the output of `cosmovisor version`. **Only** ask about bumping Cosmvisor in case the version is lower than the default Cosmovisor version set at the top. Otherwise, no need to ask this question.

Same logic applies when installing "from scratch". Even if there's a node already installed, remember the installer gives the option to install from scratch. So we should test whether in the install from scratch path, the new version of Cosmovisor needs to be installed or not.